document/js-api/index.bs

```diff
@@ -767,7 +774,15 @@ Note: Exported Functions do not have a \[[Construct]] method and thus it is not
         1. Let |ret| be ? [=Call=](|func|, undefined, |jsArguments|). If an exception is thrown, trigger a WebAssembly trap, and propagate the exception to the enclosing JavaScript.
         1. Let [|parameters|] → [|results|] be |functype|.
         1. If |results| is empty, return undefined.
-        1. Otherwise, return [=ToWebAssemblyValue=](|ret|, |results|[0]).
+        1. Otherwise, if |results| contains a single element, return [=ToWebAssemblyValue=](|ret|, |results|[0]).
+        1. Otherwise,
+            1. Let |method| be ? [=GetMethod=](|results|, @@iterator).
+            1. If |method| is undefined, [=throw=] a {{TypeError}}.
+            1. Let |values| be ? [=IterableToList=](|ret|, |method|).
+            1. Let |wasmValues| be a new, empty [=list=].
+            1. For each |value| and |resultType| in |values| and |results|,
+                1. [=Append=] [=ToWebAssemblyValue=](|value|, |resultType|) to |imports|.
+            1. Return |values|.
```

Comment:
wasmValues? However, you cannot return a list to Wasm. You'll need to convert it to a value sequence suitable for the core spec.

Reply:
Switched to wasmValues.

Do you have a suggestion for the wording for exposing a sequence to the core spec? Note that this spec already uses infra lists and Wasm sequences interchangably, as in the _args_ list in the "call an Exported Function" algorithm.

Reply:
I see, that's unfortunate. Perhaps simply introduce a meta operator ToWasmSeq that turns a list into a sequence somewhere? They cannot be interchangeable because the notion of mutation does not make sense for a syntactic sequence.

Reply:
That makes sense. I'll do this in a separate patch against the main spec repo.

Reply:
Landed the other patch, and used the same wording to fix this case.